kernel/sys/syscall.c

```diff
@@ -176,9 +176,7 @@ static int sys_sbrk(int size) {
 	spin_lock(proc->image.lock);
 	uintptr_t ret = proc->image.heap;
 	uintptr_t i_ret = ret;
-	while (ret % 0x1000) {
-		ret++;
-	}
+	ret = (ret + 0xfff) & ~0xfff; // Rounds ret to 0x1000 in O(1)
```

Comment:
This is not the right comment style. Please read the [styleguide](https://github.com/klange/toaruos/wiki/Styleguide).